akka-stream/src/main/scala/akka/stream/impl/io/TcpStages.scala

```diff
@@ -272,12 +274,6 @@ private[stream] object TcpConnectionStage {
 
       override def onUpstreamFailure(ex: Throwable): Unit = {
         if (connection != null) {
-          if (interpreter.log.isDebugEnabled) {
```

Comment:
But no, don't remove this please. This will swallow any exceptions now silently. This is how it was, and it was explicitly requested that it should *not* swallow exceptions, but log them at DEBUG. We must not revert this.